openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC1/certificate_estimation/HVAC1_PRC_variables.py

```diff
@@ -27,36 +27,51 @@ class HVAC1_input_power(Variable):
         'label': 'Rated cooling input power (kW)'
     }
 
+# class HVAC1_DNSP_Options(Enum):
```

Comment:
we should delete these variables instead of commenting as we likely won't be needing them in the future ?

Reply:
Agreed - done!